src/applications/facility-locator/containers/FacilityLocatorApp.jsx

```diff
@@ -73,6 +73,7 @@ class FacilityLocatorApp extends React.Component {
         </div>
         <DowntimeNotification
           appTitle="facility locator tool"
+          customText={{ appType: 'tool' }}
```

Comment:
This used to be hard-coded as "tool" in the [`Down` component](https://github.com/department-of-veterans-affairs/vets-website/pull/29755/files#diff-2677f12bb1836637d5a6f5d83ce66fc8b3682dff0a18b2a8de7968a330a952e8L12). `customText` is from the `formConfig` setting used by forms.